### PR TITLE
Add run button in mise.toml files

### LINKED ISSUE #68: Add run gutter icon for mise files
image

### PR DESCRIPTION
**Pull request by hverlin**
Resolves #68 

image

### CODE REVIEW

File: modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/completion/MiseConfigPsiUtils.kt [hunk 1]
```diff
@@ -92,6 +92,10 @@ fun TomlTable.parseMiseTask(): MiseTask? {
         }
     }
 
+    if (headerKey.segments.size < 2) {
+        return null
+    }
```
Comment (134130) [inline modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/completion/MiseConfigPsiUtils.kt]: How about move next to L41?

File: modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/editor/MiseRunLineMarkerProvider.kt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    private fun getTaskInfo(element: LeafPsiElement): String? {
+        val parent = element.parent ?: return null
+
+        if (parent is TomlKeySegment) {
+            // Case 1: [tasks] section
+            // [tasks]
+            // taskname = "..."
+            // other = "..."
```
Comment (134130) [inline modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/editor/MiseRunLineMarkerProvider.kt:33]: Oops, I missed this.

**Review by 134130 — APPROVED** [review]
It works perfectlly, so cool!

File: modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/completion/MiseConfigPsiUtils.kt (hunk 1 above)
Comment (hverlin) [reply modules/products/toml/src/main/kotlin/com/github/l34130/mise/toml/completion/MiseConfigPsiUtils.kt]: Yeah, we can move it, this was just a quick fix as there was a NPE

**Comment by hverlin** [comment]
@134130 I have updated the code to also show tasks from `mise tasks ls`. I believe we could cache the current mise state later on to not have to run the command every time

**Comment by 134130** [comment]
> @134130 I have updated the code to also show tasks from `mise tasks ls`. I believe we could cache the current mise state later on to not have to run the command every time

@hverlin Why not if we can! But mise is fast enough XD

**Comment by 134130** [comment]
@hverlin Thank you for your contribution!

I'll write a release note to release a new version.

**Pull request merged**